BridgeAppSDK/SBAScheduledActivityManager.swift

```diff
@@ -319,6 +319,16 @@ open class SBABaseScheduledActivityManager: NSObject, ORKTaskViewControllerDeleg
      @return                            The filtered list of activities
      */
     open func filteredSchedules(scheduledActivities: [SBBScheduledActivity]) -> [SBBScheduledActivity] {
+        if _loadingState == .fromServerWithFutureOnly {
+            // The future only will be in a state where we already have the cached data,
+            // And we will probably just be appending new data onto the cached data
+            let filteredActivities = scheduledActivities.filter({ (activity) in
```

Comment:
Consider this scenario... The user has been using their app for 2 months but hasn't used the app for a week, and they enter the app.  The cache will load 2 months of data correctly at first, and then when it loads the future from the server, the 2 months of data will be deleted and replaced with just the future data.  Then, after a long time of getting 2 months of data from the server (like 20 seconds sometimes), it will have all the correct data again. Does that make sense?  I'm trying to avoid that scenario by just merging future data loaded from the server, with the existing cached data.